### PR TITLE
Louis Goodall - Contours

### PR DESCRIPTION
**Pull request by lagoodal**

### CODE REVIEW

**Comment by wenzeslaus** [comment]
All the errors are related to JSON syntax. Try some online JSON validator. Try <https://jsonformatter.curiousconcept.com>. It seems to not only correctly describe the error from user point of view but also fix your file.

**Comment by lagoodal** [comment]
I have moved the files to the activity folder and done the JSON formatting in the link you provided and the same error appears to be there. I don't know what to do.

**Comment by petrasovaa** [comment]
> I have moved the files to the activity folder and done the JSON formatting in the link you provided and the same error appears to be there. I don't know what to do.

When you click Details on the failed check (file content/JSON), you can see:
```
AssertionError: 'Example Contributor' == 'Example Contributor' : Value of author in a task in louis_goodall.json matches value from the example file simple_example.json. author value should be different from the provided example. Change the author value in louis_goodall.json)
```

File: activities/louis_goodall.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import os
+
+import grass.script as gs
+
+
+def run_contour(scanned_elev, env, **kwargs):
+    gs.run_command(
+        "r.contour", input=scanned_elev, output="contours", levels=[15, 30, 45]
```
Comment (petrasovaa) [inline activities/louis_goodall.py]: You don't pass the env variable, see instructions in simple_example.py

File: activities/louis_goodall.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+   "tasks":[
+      {
+         "layers":[
+            [
+               "d.rast",
```
Comment (petrasovaa) [inline activities/louis_goodall.json]:
```suggestion
               "d.vect",
```

**Review by petrasovaa — CHANGES_REQUESTED** [review]
If you check the rendered result of your PR (see [instructions](https://github.com/ncsu-geoforall-lab/gis714-2022-tangible-landscape#after-opening-pr)), the image is empty because the elevation of the contours you specified is less than the elevation of the raster. It would be better to for example to dynamically choose the levels or contour step based on the elevation range. That would also make this application little bit more sophisticated than it is now.

**Comment by petrasovaa** [comment]
If you check the rendered result of your PR (see [instructions](https://github.com/ncsu-geoforall-lab/gis714-2022-tangible-landscape#after-opening-pr)), the image is empty because the elevation of the contours you specified is less than the elevation of the raster. It would be better to for example to dynamically choose the levels or contour step based on the elevation range. That would also make this application little bit more sophisticated than it is now.

**Comment by petrasovaa** [comment]
Third time's the charm!

**Pull request merged**